apps/site/pages/docs/api-extensions.mdx

```diff
@@ -3,38 +3,47 @@ import { ExtendableQueryTable } from './../../components/ExtendableQueryTable'
+2. Determine the GraphQL fragments where these fields will be used.
+3. How to consume the new fields using [New Sections](/docs/building-sections/creating-a-new-section) or [Custom Overridable Components](/docs/building-sections/overriding-components-and-props#overriding-a-native-component).
 
 ---
 
 ## Before you start
 
-Note that even though you can add information to the FastStore API schema, you must be careful not to over-fetch data on your pages. See the [best practices for fetching data on your storefront](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data).
+<Callout type="info" emoji="ℹ️">
+  Even though you can add information to the FastStore API schema, you must be careful not to over-fetch data on your pages. See the [best practices for fetching data on your storefront](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#fetch-only-the-data-you-need).
+</Callout>
 
 As stores grow, so does the possibility of using data from VTEX APIs that the FastStore API does not expose.
 Another possibility is to use data that is provided by third-party APIs. In both cases, FastStore provides these features by using API Extensions.
 
-Inside the repository, there is a clear separation about what comes from VTEX APIs and what comes from third-party APIs. First, let's take a look in the VTEX APIs Schema extension possibility.
+Then, it's necessary to define which new fields will be exposed by the FastStore API Extension.
+
+Inside the repository, there is a clear separation between what comes from VTEX APIs and what comes from third-party APIs. First, let's take a look at the VTEX APIs Schema extension possibility.
 
 ---
 
-## VTEX APIs schema extensions
+## Define VTEX APIs schema extensions
 
 FastStore streamlines the way of using data from VTEX APIs that the FastStore API does not expose.
 
-As FastStore uses GraphQL, it's required to write [type definitions](https://graphql.org/learn/schema/#object-types-and-fields) and [resolvers](https://graphql.org/learn/execution/#root-fields-resolvers) to fetch the data you need. You will learn how to do this using API Extensions by following the steps below.
+As FastStore uses GraphQL, it's required to write [type definitions](https://graphql.org/learn/schema/#object-types-and-fields) and [resolvers](https://graphql.org/learn/execution/#root-fields-resolvers) to fetch the data you need.
+You will learn how to do this using API Extensions by following the steps below.
 
 ### Folder Structure
 
-The store's repository provides the following folder structure for the VTEX APIs Schema Extensions that is held in the `src/graphql/vtex` folder.
+To declare a new type definition and resolver you need to create a directory structure like the one below. Please note that the `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path is mandatory, but you can organize the files inside those directory as you wish.
```

Comment:
```suggestion
To declare a new type definition and resolver you need to create a directory structure like the one below.

<Callout type="warning" emoji="⚠️">
Note that the `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path is mandatory, but you can organize the files inside those directory as you wish.
</Callout>
```

Reply:
Now, I'm not sure if we need this part. In line 66 we already say this, right?

Reply:
we added this Callout. As we updated the code, I ended up losing the reference for line 66 hahah, but I think they mean different things.